package.json

```diff
@@ -50,7 +50,7 @@
     "@stacks/connect-ui": "^3.0.2",
     "@stacks/network": "^1.0.2",
     "@stacks/rpc-client": "^1.0.3",
-    "@stacks/transactions": "1.3.0",
+    "@stacks/transactions": "^1.3.0",
```

Comment:
We need to keep these pinned (which is why there are conflicts), but also after you rebase the new CI will flag this